tripy/tests/frontend/ops/test_outer.py

```diff
@@ -0,0 +1,23 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from tests import helper
+import tripy as tp
+class TestOuter:
```

Comment:
Do you plan to add integration tests?

Reply:
I wasn't sure what case to test other than what's in the docstring and this frontend test.

Reply:
We should check if one of the operand is empty tensor: 
```
def test_outer_zero_size():
    vec1 = tp.Tensor([])
    vec2 = tp.arange(3, dtype=tp.float32)
    output = tp.outer(vec1, vec2)
    
    assert output.shape == (0, 3)
```

Reply:
Done, please check